conf/deployment/rosa/pullsecrete_for_nonga.yaml

```diff
@@ -0,0 +1,4 @@
+DEPLOYMENT:
```

Comment:
pullsecrete_for_nonga.yaml -> pullsecret_for_nonga.yaml

Reply:
Thank you for pointing this out. renamed file now.